app/src/main/java/com/besome/sketch/editor/PropertyActivity.java

```diff
@@ -47,15 +47,17 @@
 public class PropertyActivity extends BaseAppCompatActivity implements Kw {
 
     private final ArrayList<Integer> propertyGroups = new ArrayList<>();
-    private LinearLayout content;
     private ProjectFileBean projectFileBean;
     private ViewBean viewBean;
+    private ViewPropertyItems propertyItems;
     private boolean p;
     private String sc_id;
-    private LinearLayout r;
-    private ViewPropertyItems propertyItems;
     private int layoutPosition;
 
+    private LinearLayout content;
+    private LinearLayout layout_ads;
+    private com.besome.sketch.lib.ui.CustomScrollView scroll_view;
+    private androidx.recyclerview.widget.RecyclerView property_group_list;
```

Comment:
Can you don't do like this?
Just import class name.

Reply:
Bro, since I use an xml plugin, it transfers like this, I'll edit it next time.

Reply:
Just edit the file in Github Web.

Reply:
I edited it but there is not much difference between the old and new version. 😅